### ISSUE TITLE
Inline type hints don't update accurately

### ISSUE THREAD

**Issue by sjohnsonaz** [ionide/ionide-vscode-fsharp#1824]
### Describe the bug

In VSCode, type hints don't update accurately as code changes.

I created a function which was auto-detected to be usint `int`.  Then I used the function with `float`.  The hints in the hover showed `float`, but the inline hints still showed `int.`

### Steps to reproduce

Create the following two bindings:

``` fsharp
let line m b x = m * x + b

let y = line 1.0 1.0 1.0
```

The inline hint says:

``` fsharp
// int -> int -> int -> int
```

But the hovered hint says:
``` fsharp
val line:
   m: float ->
   b: float ->
   x: float
   -> float
```

### Expected behaviour

Both the inline hints and hovered hints should be the same.

### Screenshots

![Screenshot 2023-02-14 081742](https://user-images.githubusercontent.com/2746299/218780643-507ca6b4-6880-4af7-a617-658fb338ff22.png)

## Machine info
 - OS: Windows
 - .NET SDK version: 7.0.102
 - Ionide version: 7.4.2

**Comment by baronfel** [comment]
This seems to be resolved in 7.5.1.

**Comment by TTSKarlsson** [comment]
I still have this issue with `VSCode 1.80`  `Ionide 7.7.1` `omnisharp 1.26.0` `.NET SDK 7.0.305` on `Windows 11 Build 22621`

The error is intermittent but reoccurs relatively quickly after reload.
